epic_manager/orchestrator.py

```
@@ -365,12 +399,50 @@ async def _start_review_monitor(
+        Args:
+            instance_name: KB-LLM instance name
+
+        Returns:
+            Dictionary mapping issue_number -> pr_number
+        """
+        instance_path = Path(f"/opt/{instance_name}")
+        prs = {}
+
+        try:
+            result = subprocess.run(
+                ["gh", "pr", "list", "--json", "number,headRefName"],
```

Ruff (0.13.3)

Starting a process with a partial executable path

(S607)